test/corepool.test.ts

```diff
@@ -77,7 +77,7 @@ describe('Core Pool', () => {
 		).to.be.revertedWith('deposit not yet unlocked');
 
 		// Wait for more than a year though and...
-		const futureDate = add(start, { days: 365 });
+		const futureDate = add(start, { years: 1 });
```

Comment:
This actually needs to be 365 days, as leap years have 366 days, but solidity doesn't know that.
```suggestion
		const futureDate = add(start, { days: 365 });
```